README.md

```diff
@@ -105,3 +106,13 @@ Global Flags:
   -t, --target-url string      target URL for the attack
 
 ```
+
+## Parallelism
+
+The worker takes `-w` parameters that defines the level of parallelism used to
+produce the requests for the target. This allows us to have a request ready as
+soon as it is needed even when high volume is needed.
+
+It is important to set this parameter accurately specifically when the worker
```

Comment:
I personally wouldn't know how to set this number accurately

Reply:
Added some hints